reference/catalog/products_catalog.v3.yml

```diff
@@ -8427,11 +8429,11 @@ components:
           items:
             type: integer
         sort_order:
-          maximum: 2147483647
-          minimum: -2147483648
-          type: integer
+          type: number
```

Comment:
On the [spreadsheet ](https://docs.google.com/spreadsheets/d/1q6hXRKuVRfeq96WcEGT1lxo9aVgNmEr9WMnaFNU3OBE/edit#gid=91879601)attached to the ticket, it mentions that negative whole numbers are okay. I believe that would mean "integer" (not "number").